examples/machine-learning/a3-ultragpu-8g/nccl-tests/build-nccl-tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#SBATCH --ntasks=1
+#SBATCH --partition=a3ultra
```

Comment:
It may be worth mentioning here (or in the README) that this is a field that is dependent on the deployment setting for the partition name.